headers/functions/overlay29.h

```diff
@@ -382,6 +387,7 @@ void ConvertSecondaryTerrainToChasms(void);
 void EnsureImpassableTilesAreWalls(void);
 void InitializeTile(struct tile* tile);
 void ResetFloor(void);
+void ResetImportantSpawnPositions(struct dungeon_generation_info* gen_info);
```

Comment:
Reorder to match the symbol file, should be between `ResetInnerBoundaryTileRows` and `SpawnStairs`